sdks/python/apache_beam/runners/interactive/dataproc/dataproc_cluster_manager.py

```diff
@@ -0,0 +1,163 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pytype: skip-file
+
+import logging
+
+try:
+  from google.cloud import dataproc_v1
+except ImportError:
+  raise ImportError(
+      'Google Cloud Dataproc not supported for this execution environment.')
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class DataprocClusterManager:
+  """The DataprocClusterManager object simplifies the operations
+  required for creating and deleting Dataproc clusters for use
+  under Interactive Beam.
+  """
+  DEFAULT_NAME = 'interactive-beam-cluster'
+
+  def __init__(
+      self,
+      project_id: str = '',
```

Comment:
You can do `Optional[str]` and provide a default value `None`.

Reply:
Sounds good! Thanks!